current-federal-non-dotgov.csv

```diff
@@ -57,6 +57,8 @@ anewpostoffice.com,Federal Agency - Executive,United States Postal Service,,Wash
 any-foods.com,Federal Agency - Executive,United States Postal Service,,Washington,DC,
 any-restaurants.com,Federal Agency - Executive,United States Postal Service,,Washington,DC,
 apostoffice.com,Federal Agency - Executive,United States Postal Service,,Washington,DC,
+app.high.powerbigov.us,Federal Agency - Executive,Department of the Treasury,,Washington,DC,
+tftcistehdaf.org,Federal Agency - Executive,Department of the Treasury,,Washington,DC,
```

Comment:
Can you move `tftcistehdaf.org` so that it is also ordered alphabetically in this file?